understanding/22/focus-not-obscured-minimum.html

```diff
@@ -20,33 +21,34 @@ <h2>Focus Not Obscured Success Criterion text</h2>
         <blockquote>
             <p>When a <a>user interface component</a> receives keyboard focus, the component is not entirely hidden due to author-created content.</p>
         </blockquote>
+        <p class="note">If the interface is configurable so that the user can reposition content such as toolbars and non-modal dialogs, then only the initial positions of user-movable content would be considered for testing and conformance of this Success Criterion.</p>
      </section>
    
     <section id="intent">
         
         <h2>Intent of Focus Not Obscured</h2>
 
-        <p>For sighted people with mobility impairments who use a keyboard-like device (e.g., a switch, voice input), knowing the current point of focus is very important. Where other content can overlap with a focused item, entirely hiding the focused item means the user will not see where the focus is located.</p>
-
-        <p>Typical types of content that can overlap focused items are sticky footers, sticky headers, or non-modal dialogues. As a user tabs through the page, these layers of content can obscure the focused item, including the focus indicator. If the interface is configurable so that the user can move toolbars and non-modal dialogs around, then only the initial positions of user-movable content would be considered for testing and conformance of this success criterion.</p>
+        <p>The intent of this Success Criterion is to ensure that the item receiving keyboard focus is always visible in the user's viewport. For sighted people who rely on a keyboard (or on a device that operates through the keyboard interface, such as a switch or voice input), knowing the current point of focus is critical. The component with focus signals the interaction point on the page. Where users cannot see the item with focus, they may not know how to proceed, or may even think the system has become unresponsive.</p>
+        
+         <p>This AA criterion allows for the component receiving focus to be <em>partially</em> obscured by other author-created content. Ideally any overlap of the focused component by other content will be avoided. However, in recognition of the complex responsive designs common today, the minimum version of this requirement requires only that where other content overlaps an item receiving focus, it not entirely obscure it.</p>
```

Comment:
I'd recommend combining the paragraphs on L33 and L35 - they read a bit redundantly and the wording of the last sentence in L33 is a bit off compared to the similar but clearer wording at the end of L35 ("the minimum version of this requirement" sounds like it would be a subset of the SC this understanding article is about, rather than this SC vs a separate SC)

Reply:
I have just completed a commit for this.